### ISSUE TITLE
opposite of `emoji.emojify`

### ISSUE THREAD

**Issue by smeijer** [omnidan/node-emoji#36]
I'm looking for the opposite of `emoji.emojify`, to get messages out and into the database. Smartphone keyboards often just insert a unicode. I wish to map this to the corresponding string for database storage.

```js
// replaces all unicode emojis with the string variant,
var str = emoji.stringify('I ❤️ ☕️!');  // » "I :heart: :coffee:!"
```

**Comment by omnidan** [comment]
@smeijer that's a good idea! would you be willing to submit a PR for this? :grin:

**Comment by froatsnook** [comment]
Ask and ye shall receive.

**Comment by froatsnook** [comment]
It looks like my PR is broken for big emojis like 👩‍❤️‍💋‍👩.  It turns out I don't need this code, but I'll fix it if I have time and feel like it!

**Comment by omnidan** [comment]
implemented in #41 by @shivkanthb - thanks a lot! released `1.6.0` with this fix

**Comment by smeijer** [comment]
Sorry, I totally forgot this one. The implementation looks awesome. I only don't really like that you're now polluting the `+Object.prototype` namespace.

It could easily be fixed by replacing the method here:

https://github.com/omnidan/node-emoji/pull/41/files#diff-4a61896b6f06504368190384f1973ea0R131

```js
function getKeyByValue(obj, value) {
  for (var prop in obj) {
    if (obj[prop] === value) {
      return prop;
    }
  }
}
```

And the call:

https://github.com/omnidan/node-emoji/pull/41/files#diff-4a61896b6f06504368190384f1973ea0R150
```js
var emoji_text = getKeyByValue(Emoji.emoji, word);
```
